demo/examples/Attributes-Reserved/1_project.sysl

```diff
@@ -0,0 +1,39 @@
+# Sysl reserves certain attributes for sequence diagram generation. 
```

Comment:
Can this example be named Reserved-Attributes instead?

Reply:
haha sure sounds good!